ferrocene/doc/user-manual/src/targets/x86_64-pc-windows-msvc.rst

```diff
@@ -0,0 +1,34 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+.. _x86_64-pc-windows-msvc:
+
+:target:`x86_64-pc-windows-msvc`
+================================
+
+The ``x86_64-pc-windows-msvc`` Ferrocene target provides support for Host as
+well as Cross-compilation for Microsoft Windows on x86_64 using the native
+ABI (MSVC).
+
+Prerequisites
+-------------
+
+You need to install Microsoft Build Tools either via
+`installing Visual Studio <https://visualstudio.microsoft.com/downloads/>`_
+or installing the Build Tools directly via the same installer.
+
+Alternatively, you can `install Build Tools via winget
```

Comment:
I don't know about linking to this site -- It would be better to link to Microsoft directly or maybe not at all.